app/views/static/alert-overview-popover.html.haml

```diff
@@ -0,0 +1,10 @@
+%div
+  .tip-title
+    {{item.name}}
+  .tip-row{"ng-repeat" => "alert in vm.hoverAlerts track by $index"}
+    .tip-col.host-col{"ng-if" => "$index < 5"}
```

Comment:
hardcoding 5 alerts here? I'm *guessing* the `$index < 5` could be replaced by `! $last`, right?